apiml-common/src/main/java/com/ca/mfaas/product/service/BuildInfo.java

```diff
@@ -18,6 +20,9 @@
 @Slf4j
 public class BuildInfo {
 
+    @InjectApimlLogger
```

Comment:
BuildInfo is not a Spring component. How does it work with ```@InjectApimlLogger``` ?

Reply:
i've moved BuildInfo to common-service-core